pkg/cvo/sync_worker.go

```diff
@@ -474,6 +480,8 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 	validPayload := w.payload
 	if validPayload == nil || !equalUpdate(configv1.Update{Image: validPayload.ReleaseImage}, configv1.Update{Image: update.Image}) {
 		klog.V(4).Infof("Loading payload")
+		cvoObjectRef := &corev1.ObjectReference{APIVersion: "config.openshift.io/v1", Kind: "ClusterVersion", Name: "version"}
```

Comment:
The Event's namespace is [pulled off the referenced object][1], so this (and your test `Eventf` in 42d74a04) needs to include `Namespace` as well.

[1]: Ahh, looks like [it's pulled off the referenced object][1].

Reply:
> The Event's namespace is [pulled off the referenced object](https://github.com/kubernetes/kubernetes/blob/e6ee924f6c7973d6e864547f115a85d0c2623e2a/staging/src/k8s.io/client-go/tools/record/event.go#L360-L364), so this (and your test `Eventf` in [42d74a0](https://github.com/openshift/cluster-version-operator/commit/42d74a04bd25cd86d8a50b413413ad33be86e262)) needs to include `Namespace` as well.

Thanks.  Done